src/app/shared/components/comments-history/view-comment/view-comment.component.ts

```
@@ -194,8 +197,8 @@
       this.userComments = estatuses.filter((status) => status.us_full_name);
 
       for (let i = 0; i < this.userComments.length; i++) {
-        const prevCommentDt = dayjs(this.userComments[i - 1] && this.userComments[i - 1].st_created_at);
-        const currentCommentDt = dayjs(this.userComments[i] && this.userComments[i].st_created_at);
+        const prevCommentDt = this.dateWithTimezonePipe.transform(this.userComments[i - 1] && this.userComments[i - 1].st_created_at);
+        const currentCommentDt = this.dateWithTimezonePipe.transform(this.userComments[i] && this.userComments[i].st_created_at);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)